src/core/fr-command-rar.c

```diff
@@ -687,18 +687,10 @@ fr_command_rar_handle_error (FrCommand   *comm,
 	for (scan = g_list_last (comm->process->err.raw); scan; scan = scan->prev) {
 		char *line = scan->data;
 
-		if (strstr (line, "password incorrect") != NULL) {
-			error->type = FR_PROC_ERROR_ASK_PASSWORD;
-			break;
-		}
-
-		if (strstr (line, "password is incorrect") != NULL) {
-			error->type = FR_PROC_ERROR_ASK_PASSWORD;
-			break;
-		}
-
-		if (strstr (line, "wrong password") != NULL)
-		{
+		if ((strstr (line, "password incorrect") != NULL) ||
+		    (strstr (line, "password incorrect") != NULL) ||
```

Comment:
These lines are exactly the same. The fix for engrampa was about adding another string, namely ""Incorrect password".

Reply:
Fixed. Sorry for being so clumzy.